lib/fizzy/execute.cpp

```diff
@@ -510,6 +510,12 @@ inline uint64_t popcnt64(uint64_t value) noexcept
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
 
+__attribute__((no_sanitize("float-cast-overflow"))) inline constexpr float demote(
```

Comment:
Some comment explaining why it's needed?

Reply:
Yes, added.